.github/workflows/snap.yaml

```diff
@@ -0,0 +1,49 @@
+name: snap
+
+on:
+  # Manually trigger this from the Actions tab
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release Type'
+        required: true
+        default: 'stable'
+        type: choice
+        options:
+        - lts
+        - stable
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        release:
+        - ${{ inputs.release }}
+        platform:
+        - i386
+        - amd64
+        - armhf
```

Comment:
If you look below under the exclude section, we are actually indicating that we want to skip ARMv7 for the stable releases.

Reply:
Yeah, it's just that ARMv7 support is pretty sus in general and since it's tier 3 we don't block even LTS patch releases on having ARMv7 binaries, so I could imagine a case where we go to update the Snap for 1.6.x and it fails because the ARMv7 buildbot decided not to cooperate for that release.

Reply:
Ah, gotcha. One solution would be to try to build unconditionally in all four platform and have `continue-on-error: ${{ matrix.platform == 'armhf' }}` so it lets the rest run even if the ARMv7 one fails.

Reply:
I like that!

Reply:
Done.